inc/settings.php

```diff
@@ -25,6 +25,7 @@ class Optml_Settings {
 		'max_width'            => 2000,
 		'admin_bar_item'       => 'enabled',
 		'lazyload'             => 'disabled',
+		'scale'                => 'disabled',
```

Comment:
This should be enabled by default as right now the option is enabled as part of the lazyload.
If we disable it it will disable rescale on all active websites until the option is toggled.